java/ql/src/change-notes/2022-02-14-os-guards.md

```diff
@@ -1,6 +1,8 @@
 ---
 category: minorAnalysis
 ---
- * Add new guards `IsWindowsGuard` and `IsUnixGuard` to detect OS specific guards.
+ * Add new guards `IsWindowsGuard`, `IsSpecificWindowsVariant`, `IsUnixGuard`, and `IsSpecificUnixVariant` to detect OS specific guards.
+ * Add new predicate `getSystemProperty` that gets all expressions that retrieve system properties from a variety of sources (eg. alternative JDK API's, Google Guava, Apache Commons, Apache IO, ect..).
  * Update "Local information disclosure in a temporary directory" (`java/local-temp-file-or-directory-information-disclosure`) to remove false-positives when OS is properly used as logical guard.
+ * Update "Local information disclosure in a temporary directory" (`java/local-temp-file-or-directory-information-disclosure`) to use `getSystemProperty` to resolve more 
```

Comment:
This sentence seems to be incomplete or at least missing a period.

Reply:
Indeed, it is incomplete. My bad